http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -1291,4 +1328,30 @@ public RequestDistribution(HttpClient httpClient, Processor processor)
             });
         }
     }
+
+    private static class SecureHttpClient
+            extends HttpClient
```

Comment:
This is a workaround. We want to be able to provide a custom but faster AuthenticationStore because the default one provided by Jetty is not performant. How about renaming this to WrappedHttpClient?